### PR TITLE
[DomCrawler] Document images crawler

### PR DESCRIPTION
**Pull request by valeriangalliat**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes (symfony/symfony#13649, symfony/symfony#13650) |
| Applies to | 3.0.0 |
| Fixed tickets | symfony/symfony#12429 |

### CODE REVIEW

**Comment by xabbuh** [comment]
@valeriangalliat Thank you for creating the doc PR. FYI, I have prefixed the PR title with "WCM" (waiting code merge).

File: components/dom_crawler.rst [hunk 1]
```diff
@@ -327,6 +329,23 @@ methods to get more information about the selected link itself::
     page suffixed with ``#foo``. The return from ``getUri()`` is always a full
     URI that you can act on.
 
+Images
+~~~~~~
+
+To find an image by its ``alt`` attribute, use the ``selectImage`` method on an
+existing crawler. This returns a Crawler instance with just the selected
```
Comment (xabbuh) [inline components/dom_crawler.rst]: Please enclose "Crawler" with double backticks too.

File: components/dom_crawler.rst [hunk 2]
```diff
@@ -327,6 +329,23 @@ methods to get more information about the selected link itself::
...
+image(s). Calling ``image()`` gives you a special
+:class:`Symfony\\Component\\DomCrawler\\Image` object::
+
+    $imagesCrawler = $crawler->selectImage('Kitten');
+    $image = $imagesCrawler->image();
+
+    // or do this all at once
+    $image = $crawler->selectImage('Kitten')->image();
+
+The :class:`Symfony\\Component\\DomCrawler\\Image` object have the same
```
Comment (xabbuh) [inline components/dom_crawler.rst]: [...] object has the same [...]

File: components/dom_crawler.rst (hunk 1 above)
Comment (valeriangalliat) [inline components/dom_crawler.rst]: Shouldn't it even be a `` :class:`Symfony\\Component\\DomCrawler\\Crawler` ``?

Comment (WouterJ) [inline components/dom_crawler.rst]: I'm -1 for this, as literals always break the read flow, making text harder to read. We usually only add an API doc link once in the article.

Comment (xabbuh) [inline components/dom_crawler.rst]: We usually enclose class names with literas. Why do you not want to do this here?

**Comment by xabbuh** [comment]
@valeriangalliat Would you mind updating the description with references to the new pull requests?

**Comment by valeriangalliat** [comment]
@xabbuh Done!

**Comment by xabbuh** [comment]
Thank you!

**Comment by javiereguiluz** [comment]
Except for the minor pending comment from @xabbuh, I'm :+1: for this change. By the way, this is no longer `WCM` because the code was finally merged. Thanks @valeriangalliat.

**Comment by valeriangalliat** [comment]
@javiereguiluz Thanks, I updated the PR according to @xabbuh comment.

**Comment by xabbuh** [comment]
Thanks for not only creating this feature, but also writing the docs for it @valeriangalliat!

**Pull request merged**